src/config/ConfigManager.cpp

```diff
@@ -1780,10 +1780,14 @@ std::optional<std::string> CConfigManager::handleAnimation(const std::string& co
     PANIM->second.overridden = true;
     PANIM->second.pValues    = &PANIM->second;
 
-    // on/off
-    PANIM->second.internalEnabled = ARGS[1] == "1";
-
-    if (ARGS[1] != "0" && ARGS[1] != "1")
+    // If user specifies on/off as "on", "true", or "1" then the animation is enabled.
+    // If user specifies on/off as "off", "false", or "0", then the animation is disabled.
+    // Otherwise, we will let the user know that they specified an invalid on/off state.
+    if (ARGS[1] == "on" || ARGS[1] == "true" || ARGS[1] == "1")
```

Comment:
you can use `configStringToInt`, it handles all of these cases and more.

`PANIM->second.internalEnabled = configStringToInt(ARGS[1]);`

Reply:
Ty! Implementing this now!

Reply:
Implemented! Ty for the suggestion. Lmk if you have further thoughts. Doing some testing now.